en/docs/learn/design-api/advanced-topics/adding-an-api-state-change-workflow.md

```diff
@@ -62,8 +62,10 @@ First, enable the API state change workflow for **Approval Workflow Executor**.
 
 The elements of the above configuration are explained below.
 
-| Element name                                       | Description                                                                                                                                               |
-|----------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|                                                                f                                                                  |
-| `stateList`| Mandatory          | This is a comma separated list of the current state and intended action. For example, Created:Publish,Published:Block 
-
-                                                                |
+| Element Name               | Mandatory/Optional | Description                                                                                                                                                                                         |
+|----------------------------|--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `processDefinitionKey`     | Mandatory          | BPMN process definition ID.  BPMN process provided with AM as default has `APIStateChangeApprovalProcess` as the ID                                                                                 |
```

Comment:
I don't see this in the configuration.  where is it?

Reply:
These configurations will be used in step 3.